.editorconfig

```diff
@@ -151,5 +151,8 @@ dotnet_diagnostic.CA1827.severity = suggestion
 dotnet_diagnostic.CA1829.severity = suggestion
 # CA1834: Consider using 'StringBuilder.Append(char)' when applicable
 dotnet_diagnostic.CA1834.severity = suggestion
+# CA1845: Use span-based 'string.Concat'
```

Comment:
FYI @captainsafia I made a small change to your PR

Reply:
Ah I see -- didn't realize these were meant to be ordered.